gui/default/syncthing/core/connectivityStatusModalView.html

```diff
@@ -0,0 +1,54 @@
+<modal id="connectivity-status" status="{{connectivityStatusParams.status}}" icon="fas fa-fw {{connectivityStatusParams.type == 'listeners' ? 'fa-sitemap' : 'fa-map-signs'}}" heading="{{connectivityStatusParams.heading}}" large="no" closeable="yes">
+  <div class="modal-body" ng-switch="connectivityStatusParams.type">
+    <div ng-switch-when="listeners">
+      <div ng-if="listenersRunning.length > 0">
```

Comment:
Maybe display some text for `== 0`, saying that no listeners are active and connections can only be established by this device dialing others (or maybe leave out the second bit, might not be helpful/understandable).  
Similar thing for discovery.

Reply:
Good idea. Added for both listeners and discovery.

Reply:
Shouldn't the condition also include failed ones, i.e. running + failed == 0?

Reply:
That would show it only when no methods are *configured* at all.  I think the text also applies when none are running because all configured ones have failed.